machines/sns57.nix

```diff
@@ -23,6 +23,7 @@ in {
 
   users.users.theano = {
     isNormalUser = true;
+    extraGroups = [ "wheel" ];	
```

Comment:
For QEMU, you almost certainly do not need `wheel` access, you probably need `kvm`. I suggest adding both, for now, so you're not blocked, but avoid running things with `sudo` unless you actually have to.